template/README.rst

```diff
@@ -0,0 +1,40 @@
+<demo_name> Demo
+=============
+Short blurb on what this demo does.
+
+Usage
+-----
+A simple command on how to run the demo. For example,
+::
+  python <demo_name>.py
+
+Code Overview
+-------------
+Give a general overview of how the code works.
+
+When possible, use bullet points so that:
+
+* Ideas are bite-sized
+* Ideas are quickly conveyed
+
+Code Specifics
+--------------
+Draw attention to certain parts of the code implementation.
+
+This is the place to:
+
+* Highlight an implementation
+* Talk about unusual or potentially difficult parts of the code
```

Comment:
The 1st bullet is not clear to me. Are you thinking about situations where the developer has chosen one of many possible algos?

Reply:
Yes, I was thinking along those lines. If there is something particularly clever in the code, it should be highlighted in the readme.